src/js/menu/menu-item.js

```diff
@@ -63,11 +64,21 @@ class MenuItem extends ClickableComponent {
     // The control is textual, not just an icon
     this.nonIconControl = true;
 
-    return super.createEl('li', assign({
+    const el = super.createEl('li', assign({
       className: 'vjs-menu-item',
-      innerHTML: `<span class="vjs-menu-item-text">${this.localize(this.options_.label)}</span>`,
       tabIndex: -1
     }, props), attrs);
+
+    while (el.firstChild) {
+      el.removeChild(el.firstChild);
+    }
+
+    el.appendChild(createEl('span', {
+      className: 'vjs-menu-item-text',
+      textContent: this.localize(this.options_.label)
+    }));
```

Comment:
to do less work, we potentially can replace the first child with this new span and potentially do less work
```js
el.replaceChild(createEl('span', {
  className: 'vjs-menu-item-text',
  textContent: this.localize(this.options_.label)
}, el.firstChild)
```
Based on the output of ClickableComponent and the expected output of MenuItem, that's the only change.

Reply:
Alternatively, we leave as is, but we should create an issue for this to do a follow-up refactor where we look and see where we can reduce the amount of work we're doing when creating components.